.idea/.gitignore

```diff
@@ -0,0 +1,9 @@
+# generated by IntelliJ IDEA
+/shelf/
+/workspace.xml
+/compiler.xml
+/gradle.xml
+/jarRepositories.xml
+/misc.xml
+/runConfigurations.xml
+/uiDesigner.xml
```

Comment:
Could you add `discord.xml` to this as well? It's from a plugin I use

Reply:
Ideally, such gitignore rules should go into your local `$HOME/.config/git/ignore`. For details, see https://git-scm.com/docs/gitignore

Reply:
Could you put it in the project gitignore?
I'm just lazy :P

Reply:
Done in 3fd227935449cdc9cbef0fb3ba66e412fcccebd5.

I strongly recommend using the `$HOME/.config/git/ignore` file though – that way _all_ projects that you edit in IntelliJ IDEA will have the file ignored. Using in-repository `.gitignore` file for all plugins can be cumbersome and gets out of hand easily when more people are involved. For example, I have "JPA Buddy" plugin installed that also puts its own file in `.idea` directory, but it's not a good idea for me to gitignore it in [every Java project](https://github.com/rybak?tab=repositories&q=&language=java&sort=) that I touched, especially because it is completely irrelevant for most of them.

Reply:
Just to add context: `uiDesigner.xml` is there, because it's one of the so called "bundled" plugins, i.e. every default installation of IntelliJ IDEA has it.

![image](https://user-images.githubusercontent.com/624072/160291297-b89b7652-83c5-4d77-a46b-ddcaa464072e.png)

Reply:
Yeah I was aware